Tasks/artificial_rain.rb

```diff
@@ -0,0 +1,23 @@
+# Return the maximal number of watered sections if we create artificial rain
+# above exactly one section.
+# garden - the array with section's height
+# exp [4, 2, 3, 3, 2] return 4. The most optimal choice will be either of the
+# sections with the height of 3 because the water will flow to its neighbours
+# covering 4 sections altogether: 2, 3, 3, 2.
+
+def artificial_rain(garden)
+  max_count = low_high = 0
```

Comment:
Try to not do so, In case you will work with some objects it may produce unexpected behaviour(changing variable1 will also change variable2)